core/src/wallet/common/Operation.h

```diff
@@ -68,6 +69,7 @@ namespace ledger {
             Option<EthereumLikeBlockchainExplorerTransaction> ethereumTransaction;
             Option<RippleLikeBlockchainExplorerTransaction> rippleTransaction;
             Option<TezosLikeBlockchainExplorerTransaction> tezosTransaction;
+            Option<stellar::Operation> stellarOperation;
```

Comment:
Because the structure of stellar is a bit different.

Most coin have one tx -> multiple ledger operations
A stellar transaction is a bag of stellar operations which are not ledger operations
tx -> multiple stellar operations -> each operation can create multiple ledger operation.